Automattic/wp-calypso
# Stats: Replace Yoast SEO promo with Jetpack mobile app promo

#### Proposed Changes

Adds the Jetpack mobile app promo and removes the Yoast SEO one.

Before:

![APromoSEO](https://user-images.githubusercontent.com/40267301/202101081-db395e83-8d1a-4208-8bc1-8654833efe99.png)

After:

![APromoJetpackMobileApp](https://user-images.githubusercontent.com/40267301/202101110-615ef3f2-0750-4d8e-b13b-b3f6daa36a27.png)

Currently just updating the Stats page. The designs call for the same promo card on the Insights and Ads pages but it looks overpowering there with the older page layout. I'm thinking we should wait until we "grid" those layouts before adding this component in.

#### Testing Instructions

1. Visit the Calypso Live link.
2. Visit the Stats page.
3. Scroll down and confirm the new promo card is present.
4. Confirm it matches screenshot.
5. Confirm it behaves appropriately on smaller screen sizes.



#### Pre-merge Checklist





Related to #69230. Depends on #70058.
